python/fledge/services/core/api/service.py

```diff
@@ -454,8 +499,26 @@ async def update_service(request: web.Request) -> web.Response:
         if name not in installed_services:
             raise KeyError("{} service is not installed yet. Hence update is not possible.".format(name))
 
+        # Check Pre-conditions from Packages table
```

Comment:
do we check if already up to date to newest version or try update anyway?

Reply:
As we do not have the version info details when installed initially. Therefore due to unavailability of previous version we always try to update and if update with same version it returns "Nothing to update"

And in FL world there is NO such update  scenario exists; as we are releasing all plugins/service with same bumped versions.

Reply:
OK